pkg/controllers/clusterresourceplacement/controller.go

```diff
@@ -713,7 +774,7 @@ func (r *Reconciler) buildPlacementStatus(crp *fleetv1beta1.ClusterResourcePlace
 		appliedCondition := metav1.Condition{
 			Status:             metav1.ConditionUnknown,
 			Type:               string(fleetv1beta1.ClusterResourcePlacementAppliedConditionType),
-			Reason:             ApplyPendingReason,
+			Reason:             ApplyNotNeededReason,
```

Comment:
Emm, to me `pending` is good enough already.

Reply:
@ryanzhang-oss what's your opinion about this? Do you think if we should distinguish the ApplyNotNeeded or ApplyPending cases?

i think it will be helpful.

Reply:
I can see both sides of the story.  Pending could be interpreted as just an unknown. Since we already have an "unknown" status, I guess it doesn't hurt to provide a bit more information. The only downside it that we need to make sure we don't confuse the cx.